client/my-sites/customer-home/cards/features/grow-earn/index.jsx

```diff
@@ -0,0 +1,90 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { Card } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import CardHeading from 'components/card-heading';
+import VerticalNav from 'components/vertical-nav';
+import VerticalNavItem from 'components/vertical-nav/item';
+import { getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
+import { SIDEBAR_SECTION_TOOLS } from 'my-sites/sidebar/constants';
+import { expandMySitesSidebarSection as expandSection } from 'state/my-sites/sidebar/actions';
+import { bumpStat, composeAnalytics, recordTracksEvent } from 'state/analytics/actions';
+import { getSelectedEditor } from 'state/selectors/get-selected-editor';
+import { getSiteOption } from 'state/sites/selectors';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+export const GrowEarn = ( { siteSlug, expandToolsAndTrack } ) => {
+	const translate = useTranslate();
+
+	return (
+		<Card className="grow-earn">
+			<CardHeading>{ translate( 'Grow & Earn' ) }</CardHeading>
+			<h6 className="grow-earn__card-subheader">
+				{ translate( 'Grow your audience and earn money' ) }
+			</h6>
+			<VerticalNav>
+				<VerticalNavItem
+					path={ `/marketing/connections/${ siteSlug }` }
+					onClick={ () => expandToolsAndTrack( 'earn', 'share_site' ) }
+				>
+					{ translate( 'Share my site' ) }
+				</VerticalNavItem>
+				<VerticalNavItem
+					path={ `/marketing/tools/${ siteSlug }` }
+					onClick={ () => expandToolsAndTrack( 'earn', 'grow_audience' ) }
+				>
+					{ translate( 'Grow my audience' ) }
+				</VerticalNavItem>
+				<VerticalNavItem
+					path={ `/earn/${ siteSlug }` }
+					onClick={ () => expandToolsAndTrack( 'earn', 'money' ) }
+				>
+					{ translate( 'Earn money' ) }
+				</VerticalNavItem>
+			</VerticalNav>
+		</Card>
+	);
+};
+
+export default connect(
+	state => {
+		const siteId = getSelectedSiteId( state );
+		const isClassicEditor = getSelectedEditor( state, siteId ) === 'classic';
+		const pageOnFront = 'page' === getSiteOption( state, siteId, 'show_on_front' );
+		return {
+			siteSlug: getSelectedSiteSlug( state ),
+			isStaticHomePage: ! isClassicEditor && pageOnFront,
+		};
+	},
+	dispatch => ( {
+		trackAction: ( section, action, isStaticHomePage ) =>
+			dispatch(
+				composeAnalytics(
+					recordTracksEvent( `calypso_customer_home_${ section }_${ action }_click`, {
+						is_static_home_page: isStaticHomePage,
+					} ),
+					bumpStat( 'calypso_customer_home', `${ section }_${ action }` )
+				)
+			),
+		expandToolsSection: () => dispatch( expandSection( SIDEBAR_SECTION_TOOLS ) ),
+	} ),
+	( stateProps, dispatchProps, ownProps ) => ( {
+		...stateProps,
+		...ownProps,
+		expandToolsAndTrack: ( section, action ) => {
+			dispatchProps.expandToolsSection();
+			dispatchProps.trackAction( section, action, stateProps.isStaticHomePage );
```

Comment:
Whoops I got the ordering wrong in other spots. The default method is
https://github.com/reduxjs/react-redux/blob/172b2334d3a1240248c049a40b88a845bd0ecf5d/src/connect/mergeProps.js#L4

Reply:
e1ba3e2eff